pilot/pkg/networking/plugin/authz/test_helper.go

```diff
@@ -192,6 +192,13 @@ func generatePolicyWithHTTPMethodAndGroupClaim(methodName, claimName string) *po
 
 // nolint:deadcode
 func generateExpectRBACForSinglePolicy(serviceRoleName string, rbacPolicy *policy.Policy) *policy.RBAC {
+	// If |serviceRoleName| is empty, which means the current service does not have any matched ServiceRoles.
```

Comment:
This comment is a bit confusing, in the context of `generateExpectRBACForSinglePolicy()`, there is no concept of service, so what does it mean `the current service does not have any matched ServiceRoles.`?

Reply:
For example, we can have many ServiceRoles, but none of them is applied to service A. When we don't have any matching ServiceRole for service A, we simply return an empty RBAC policy.